index.html

```diff
@@ -0,0 +1,348 @@
+                  <use href="./images/sprite.svg#icon-group"></use>
+                </svg>
+              </li>
+              <li class="item">
+                <svg width="40" height="52">
+                  <use href="./images/sprite.svg#icon-group-1"></use>
+                </svg>
+              </li>
+              <li class="item">
+                <svg width="43.46" height="41.18">
+                  <use href="./images/sprite.svg#icon-group-2"></use>
+                </svg>
+              </li>
+              <li class="item">
+                <svg width="84.44" height="40.62">
+                  <use href="./images/sprite.svg#icon-group-3"></use>
+                </svg>
+              </li>
+              <li class="item">
+                <svg width="62.54" height="45.3">
+                  <use href="./images/sprite.svg#icon-group-4"></use>
+                </svg>
+              </li>
+              <li class="item">
+                <svg width="93.79" height="43.93">
+                  <use href="./images/sprite.svg#icon-group-5"></use>
+                </svg>
+              </li>
+            </ul>
+          </div>
+        </section>
+      </div> 
+    </main>
+   <!-- close main -->
+  
+   <!-- open footer -->
+  <footer class="footer">
+    <address>
+      <div class="container adress"> 
+        <div class="contact-footer">
+          <a href="index.html" class="logo-footer"><span class="logo-web">Web</span>Studio</a>
+          <ul class="list">
+            <li>г. Киев, пр-т Леси Украинки, 26</li>
+            <li><a href="mailto:info@example.com" class="link">info@example.com</a></li>
+            <li><a href="tel:+380991111111" class="link">+38 099 111 11 11</a></li>
+          </ul>
+        </div>
+        <div clas="social-footer">
+          <div class="join">Присоединяйтесь</div>
+          <ul class="list-icon social">
+            <li class="icon">
+              <svg width="20" height="20">
+                <use href="./images/symbol-defs.svg#icon-instagram"></use>
+              </svg>
+            </li>
+            <li class="icon">
+              <svg width="20" height="16.25">
+                <use href="./images/symbol-defs.svg#icon-twitter"></use>
+              </svg>
+            </li>
+            <li class="icon">
+              <svg width="20" height="20">
+                <use href="./images/symbol-defs.svg#icon-facebook"></use>
+              </svg>
+            </li>
+            <li class="icon">
+              <svg width="20" height="20">
+                <use href="./images/symbol-defs.svg#icon-linkedin"></use>
+              </svg>
+            </li>
+          </ul>
+        </div>
+          <form class="form-mail">
+            <span class="follow-title">Подпишись на рассылку</span>
+            <div class="wrap-follow">
+              <label class="lable-mail">
+                <input class="input-mail" type="email" name="" id="" placeholder=" ">
```

Comment:
name and id empty. Fix that. Placeholder is empty as well. Why?

Reply:
placeholder not empty there is space. name and id delete